### PR TITLE
[Converter] Change external converter param @open sesame 04/22 19:59

### PR DESCRIPTION
**Pull request by gichan-jang**
Change external converter param
 - Remove unnecessary param (frames_in, frame_size)

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: ext/nnstreamer/extra/nnstreamer_protobuf.h [hunk 1]
```diff
@@ -39,16 +39,14 @@ gst_tensor_decoder_protobuf (const GstTensorsConfig * config,
     const GstTensorMemory * input, GstBuffer * outbuf);
 
 /**
- * @brief tensor converter plugin's NNStreamerExternalConverter callback 
+ * @brief tensor converter plugin's NNStreamerExternalConverter callback
+ * @param[in] priv private data of the external converter
```
Comment (dongju-chae) [inline ext/nnstreamer/extra/nnstreamer_protobuf.h:43]: Please check the doxygen tag again.

Comment (gichan-jang) [reply ext/nnstreamer/extra/nnstreamer_protobuf.h]: Thanks. Updated.

**Review by jaeyun-jung — APPROVED** [review]

File: ext/nnstreamer/extra/nnstreamer_protobuf.cc [hunk 2]
```diff
@@ -144,8 +143,7 @@ gst_tensor_converter_protobuf (GstBuffer *in_buf, gsize *frame_size,
   fr = tensors.mutable_fr ();
   config->rate_n = fr->rate_n ();
   config->rate_d = fr->rate_d ();
-  *frames_in = 1;
-  *frame_size = 0;
+
```
Comment (dongju-chae) [inline ext/nnstreamer/extra/nnstreamer_protobuf.cc:146]: Intended?

File: ext/nnstreamer/extra/nnstreamer_protobuf.h [hunk 3]
```diff
@@ -39,16 +39,13 @@ gst_tensor_decoder_protobuf (const GstTensorsConfig * config,
     const GstTensorMemory * input, GstBuffer * outbuf);
 
 /**
- * @brief tensor converter plugin's NNStreamerExternalConverter callback 
+ * @brief tensor converter plugin's NNStreamerExternalConverter callback
  * @param[in] buf The input stream buffer
```
Comment (dongju-chae) [inline ext/nnstreamer/extra/nnstreamer_protobuf.h:43]: Let's match variable name

File: ext/nnstreamer/extra/nnstreamer_protobuf.cc (hunk 2 above)
Comment (gichan-jang) [reply ext/nnstreamer/extra/nnstreamer_protobuf.cc]: Removed. Thanks!

File: ext/nnstreamer/extra/nnstreamer_protobuf.h (hunk 3 above)
Comment (gichan-jang) [reply ext/nnstreamer/extra/nnstreamer_protobuf.h]: Updated.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**